### DISCUSSION TITLE
Possibility of Integration of Open Web UI to External Application through Web API

### DISCUSSION THREAD

**Discussion by Urvesh71** [open-webui/open-webui#6876] [General]
Goal: I want to integrate Open Web UI to SAP Database Tables so that I can do Question - Answer to the SAP Database. To do this, I have created a CDS Views of one table followed by OData Service and at the end, I got an Web API to retrieve the data from that one table through CDS Views. 

Now I would like to intereact with this same data through Open Web UI. Does it give any possibility to connect the data through end point?
If yes, Please guide me how?

**Comment by git-dev-admin** [comment]
hello @Urvesh71 , I am interested to know if you were able to do it?
